sklearn/neural_network/tests/test_mlp.py

```diff
@@ -266,6 +266,32 @@ def test_lbfgs_regression():
             # Non linear models perform much better than linear bottleneck:
             assert_greater(mlp.score(X, y), 0.95)
 
+def test_lbfgs_maxfun():
+    # Test lbfgs parameter max_fun
+    # It should independently limit the 
+    # number of iterations for lbfgs
+    max_fun = 10
+    expected_num_iter = 10 
```

Comment:
add `#  classification tests: `

Reply:
Maybe I don't understand what you mean by classification tests... but the test already has both mlp-regression and mlp-classification tests in it.. right?
Or do you mean, I need to check the quality of the regression/classification? (this is already done in other tests)

Reply:
ohh... i see you just want a comment delineating the different tests.. got it.